test/extensions/filters/network/thrift_proxy/router_test.cc

```diff
@@ -1751,7 +1756,17 @@ TEST_F(ThriftRouterTest, UpstreamDraining) {
 TEST_F(ThriftRouterTest, UpstreamPartialResponse) {
   initializeRouter();
 
-  EXPECT_CALL(callbacks_, resetDownstreamConnection());
+  // EXPECT_CALL(callbacks_, resetDownstreamConnection());
```

Comment:
Is there a test that invokes the underflow path?

Reply:
thanks for catch the leftover. yes there is. 

https://github.com/envoyproxy/envoy/blob/c91181872158622de0dadd154d7131c815cdd81d/test/extensions/filters/network/thrift_proxy/router_test.cc#L954